examples/imagenet/input_pipeline_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for flax.examples.imagenet.input_pipeline."""
+
+import os
+import tempfile
+
+from absl import logging
+from absl.testing import absltest
+
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets as tfds
+
+import input_pipeline
+
+
+class InputPipelineTest(absltest.TestCase):
+
+  def test_train_and_evaluate(self):
+    # Get testing directory.
+    tf.enable_v2_behavior()
```

Comment:
not needed. I think we don't support TF1 in Flax.

Reply:
Done! Thanks for pointing this out.